src/git/NOTES.md

```diff
@@ -2,6 +2,6 @@
 
 ## OS Support
 
-This Feature should work on recent versions of Debian/Ubuntu, RedHat Enterprise Linux, Fedora, Alma, and RockyLinux distributions with the `apt`, `yum`, `dnf`, or `microdnf` package manager installed.
+This Feature should work on recent versions of Alpine, Debian/Ubuntu, RedHat Enterprise Linux, Fedora, Alma, and RockyLinux distributions with the `apk`, `apt`, `yum`, `dnf`, or `microdnf` package manager installed.
```

Comment:
This is making it sound like Alpine works for all Feature options, however, build from source would fail for alpine.

See https://github.com/devcontainers/features/blob/4a2d927a8ffb549236cb129201da2baf9423f8ec/src/git/install.sh#L235

Can we also ensure Alpine is supported in that case? Else, we'd need to update OS Support to make it super clear.

Reply:
Building from source is a harder problem on Alpine because it's musl vs. glibc. I'm working on an implementation but I wanted to provide a working partial implementation based on the package manager, since the `apk` package is kept up to date. I'll keep working on the build from source.